assets/js/DateConversion.js

```diff
@@ -0,0 +1,41 @@
+
+export default {
```

Comment:
for simplicity, I think you could change this to be `export default function (date, precision) {`

followed by the body of the function.

You can then import this as import getJulianDate from 'assets/js/DateConversion.js'